pkg/driver/controller.go

```diff
@@ -99,8 +100,35 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 		}
 	}
 
-	volumeParams := req.GetParameters()
-	fsType := volumeParams[FsTypeKey]
+	var (
+		fsType      string
+		volumeType  string
+		iopsPerGB   int
+		isEncrypted bool
+		kmsKeyId    string
+	)
+
+	for key, value := range req.GetParameters() {
+		switch strings.ToLower(key) {
```

Comment:
Just to clarify, do we support a lower case `fsType` in-tree?  What about the value?

Reply:
We do support lower case `fstype` in-tree as the key, that's [this PR](https://github.com/kubernetes/kubernetes/issues/75941) fixes for in-tree.